.gitignore

```diff
@@ -1,9 +1,17 @@
+
 /.cache
 /.work
 /_output
 cover.out
 /vendor
 /.vendor-new
+.registry
```

Comment:
This can be dropped as it is from legacy crossplane logic :)

Reply:
thanks for the review Dan!
PTAL